_example/simple/simple.go

```diff
@@ -35,20 +41,36 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	defer stmt.Close()
+	defer func() {
+		if err := stmt.Close(); err != nil {
+			log.Fatal(err)
+		}
+	}()
 	for i := 0; i < 100; i++ {
 		_, err = stmt.Exec(i, fmt.Sprintf("こんにちわ世界%03d", i))
 		if err != nil {
 			log.Fatal(err)
 		}
 	}
-	tx.Commit()
+	errCommit := tx.Commit()
+	defer func() {
+		if err := tx.Rollback(); err != nil {
```

Comment:
This should be in a `defer` right after `db.Begin()` (and its err check), since we should do it if we fail at any point during the transaction.